test/test_utils.py

```diff
@@ -0,0 +1,11 @@
+from opimodel import utils
+import pytest
+
+
+@pytest.mark.parametrize('input,output',
+                         [('hello', 'HELLO'),
+                          ('Hello:', 'HELLO_'),
+                          ('Hello: Goodbye', 'HELLO_GOODBYE'),
+                          ('Dummy 1', 'DUMMY_1')])
+def test_mangle_name_handles_various_cases(input, output):
```

Comment:
You only know that because you wrote it. As `_` is identified a 'special' character (because non [A-Za-z0-9] characters gets converted into one) it would be worth having a test that demonstrates how it is handled.